### ISSUE TITLE
perworldinventory wiping inventories

### ISSUE THREAD

**Issue by aycgit** [Gnat008/PerWorldInventory#86]
my server is 1.10 and runs pwi 1.7.6(im trying to update now)
issue is as the title says the inventories are or, fully wiped or the inventory stays but u have 0 of every item, this only happens sometimes, debugging is hard because it doesnt happen all the time
no errors, this is very irrittating because i run a faction server and inventories getting reset will mean items will be lost, im using a plugin that automaticly tps u on join to the hub

**Comment by Gnat008** [comment]
What other plugins do you have?

**Comment by aycgit** [comment]
(just saying possible relevants) multiverse, tplogin(teleports u on join) ultracosmetics(clears inventory for hub world but inventory should always be almost empty

**Comment by aycgit** [comment]
![2016-08-25_15 33 07](https://cloud.githubusercontent.com/assets/13834197/17971015/45a9a184-6ad9-11e6-8289-a04f1a2fc8c6.png)
this time it didnt fully whipe my inventory but u see what i mean, happens when changing world
could it be that this is caused by pwi not having enough time to write data to disk?

**Comment by Gnat008** [comment]
That's a new one... I wouldn't think so. Could you send me the data file from a player after the issue occurs?

**Comment by aycgit** [comment]
![2016-08-31_20 09 13](https://cloud.githubusercontent.com/assets/13834197/18140693/ef7280fa-6fb6-11e6-913a-77ce314f5bb7.png) it happened again

the only playerdata folder i could find was empty, where can i find the file u need?

**Comment by aycgit** [comment]
found the file!
http://pastebin.com/wUHRVxqd

**Comment by aycgit** [comment]
bump, im still here

**Comment by J2D22** [comment]
Bump, same issue, openned a new ticket though

**Comment by aycgit** [comment]
Any progress? My players are hating this,

**Comment by J2D22** [comment]
No progress at all, still waiting on the developper to comment on this.

**Comment by Gnat008** [comment]
Moving is an ongoing process still. Could you please see Issue #115, I posted a PWI version that should give us more information on this issue.

**Comment by Gnat008** [comment]
If this is the same issue as #115, it should now be fixed.

**Comment by aycgit** [comment]
@Gnat008 not the same, We (atleast i) didnt get any errors

**Comment by Gnat008** [comment]
Is it still occurring with the latest version?

**Comment by MapleMoosey** [comment]
I am currently using the current version and we are a 1.11 server and it has wiped all the ender chest inventories of all the players.
